pkg/api/request.go

```diff
@@ -311,7 +311,7 @@ func (a *API) RevokeRequest(w http.ResponseWriter, r *http.Request, requestID st
 		apio.Error(ctx, w, err)
 		return
 	}
-	apio.JSON(ctx, w, res, http.StatusOK)
+	apio.JSON(ctx, w, nil, http.StatusOK)
```

Comment:
Is this change intentional?

Reply:
Yes, I was reviewing th eAPI responses to check they matched the openapi spec, this one was returning a raw request when the spec states no response body.

I found some other instances where .ToAPI() was missing and fixed those up too.

This api probably should return something, but I figure it should at least match the schema!

Reply:
Good call!